packages/cardpay-sdk/sdk/native-currencies.ts

```diff
@@ -11,6 +11,50 @@ export default {
     smallThreshold: 0.003,
     symbol: 'Ξ',
   },
+  DAI: {
+    alignment: 'left',
+    assetLimit: 0.001,
+    currency: 'DAI',
+    decimals: 18,
+    label: 'Dai',
+    mask: '[09999999999]{.}[999999999999999999]',
+    placeholder: '0',
+    smallThreshold: 0.003,
+    symbol: '',
+  },
+  CARD: {
+    alignment: 'left',
+    assetLimit: 0.001,
+    currency: 'CARD',
+    decimals: 18,
+    label: 'CARD',
+    mask: '[09999999999]{.}[999999999999999999]',
+    placeholder: '0',
+    smallThreshold: 0.003,
```

Comment:
A CARD token is very much smaller than an ETH token in value. It's much more similar to USD in terms of value. I think a small threshold for this can be 1.